ui/packages/consul-ui/app/components/consul/lock-session/form/index.hbs

```diff
@@ -32,7 +32,7 @@
           <dd>{{or api.data.TTL '-'}}</dd>
           <dt>Behavior</dt>
           <dd>{{api.data.Behavior}}</dd>
-{{#let (union api.data.NodeChecks api.data.ServiceChecks) as |checks|}}
+{{#let api.data.checks as |checks|}}
```

Comment:
You may want to update this to include `@` or `this.` ahead of the variable, to be future-compatible with Octane.

Reply:
@randallmorey I tried to make this change but LockSession form, KV form, and Intention form are all still using the `'@ember/component'`. I tried just update the js file to use `'@glimmer/component'` but that broken the app. A lot more refactoring required to make this work.

Reply:
Understood.  No worries.